mock/remote.go

```diff
@@ -0,0 +1,204 @@
+package mock
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/sirupsen/logrus"
+
+	"github.com/gostor/gotgt/pkg/api"
+	"github.com/gostor/gotgt/pkg/config" /* init lib */
+	"github.com/gostor/gotgt/pkg/port/iscsit"
+	"github.com/gostor/gotgt/pkg/scsi"
+	_ "github.com/gostor/gotgt/pkg/scsi/backingstore" /* init lib */
+	"github.com/gostor/gotgt/pkg/scsi/backingstore/remote"
+)
+
+type remoteBs struct {
+	Volume     string
+	Size       int64
+	SectorSize int
+
+	isUp bool
+	rw   api.RemoteBackingStore
+
+	tgtName      string
+	lhbsName     string
+	clusterIP    string
+	cfg          *config.Config
+	targetDriver scsi.SCSITargetDriver
+	stats        scsi.Stats
+}
+
+func (r *remoteBs) ReadAt(data []byte, size int64) (int, error) {
+	return 0, nil
+}
+
+func (r *remoteBs) WriteAt(data []byte, size int64) (int, error) {
+	return 0, nil
+}
+
+func (r *remoteBs) Sync() (int, error) {
+	return 0, nil
+}
+
+func (r *remoteBs) Unmap(bs int64, size int64) (int, error) {
+	return 0, nil
+}
+
+func initializeSCSITarget(size int64) {
+	iscsit.EnableStats = true
+	scsi.SCSIVendorID = "OPENEBS"
+	scsi.SCSIProductID = "JIVA"
+	scsi.SCSIID = "iqn.2016-09.com.jiva.openebs:iscsi-tgt"
```

Comment:
would u please replace `jiva.openebs` to `gotgt.gostor`? thanks.

Reply:
ok